src/viur/core/bones/relational.py

```diff
@@ -923,8 +869,10 @@ def buildDBSort(
         origFilter = dbFilter.queries
         if origFilter is None or not "orderby" in rawFilter:  # This query is unsatisfiable or not sorted
             return dbFilter
-        if "orderby" in rawFilter and isinstance(rawFilter["orderby"], str) and rawFilter["orderby"].startswith(
-                f"{name}."):
+        if (
+            "orderby" in rawFilter and isinstance(rawFilter["orderby"], str)
+            and rawFilter["orderby"].startswith(f"{name}.")
+        ):
```

Comment:
Can you please keep the ugly code, as it is refactored by #1077 already for viur-core 3.7 and results in a merge conflict.

```suggestion
        if "orderby" in rawFilter and isinstance(rawFilter["orderby"], str) and rawFilter["orderby"].startswith(
                f"{name}."):
```

Reply:
well okay. The linter always disagrees with PyCharms, so I'll probably have to mediate manually again.

Reply:
well okay, then leave it as is, I'll manually resolve the conflict.